classes/sota_minnowboard.bbclass

```diff
@@ -4,4 +4,7 @@ EFI_PROVIDER_sota = "grub-efi"
 WKS_FILE_sota = "efiimage-sota.wks"
 IMAGE_BOOT_FILES_sota = ""
 
+IMAGE_FSTYPES_remove_sota = "live hddimg"
 OSTREE_KERNEL_ARGS ?= "ramdisk_size=16384 rw rootfstype=ext4 rootwait rootdelay=2 console=ttyS0,115200 console=tty0"
+
+IMAGE_INSTALL_append = " minnowboard-efi-startup"
```

Comment:
Is this correct?  I'm getting a "ERROR: Nothing RPROVIDES 'minnowboard-efi-startup' (but /home/phil/pro/poky/meta/recipes-core/images/core-image-minimal.bb RDEPENDS on or otherwise requires it)" running oe-selftest -r updater

Reply:
Ooops. No, sorry, forgot to merge some stuff from pyro in meta-updater-minnoboard. I'll check other machine layers as well.